src/libs/masq.js

```diff
@@ -84,25 +84,18 @@ export default class MasqStore {
 
   async getAllPois() {
     await this.checkInit()
-    const list = await this.masq.list()
```

Comment:
list need prefix param, you take a risk passing undefined

Reply:
prefix is optional

Reply:
it's error prone : you should use as you want another method listPrefix or an optionnal defaulted to {} field  which could contain prefix